### DISCUSSION TITLE
[FeatureRequest] Variables: RFC3339/ISO8601 formatting option to __from & __ to variables

### DISCUSSION THREAD

**Discussion by tinproject** [grafana/grafana#38349] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

An advance formatting option to transform `__from` & `__to` variables values from timestamps to RFC3339/ISO8601 timestamps.

E.g.: `${__from:rfc3339}`

**Why is this needed**:

I'm trying to create a direct link from a Grafana Dashboard to Cloudwatch Logs so you can go directly to the relevant logs for the same time span as the dashboard.

Example:
- Grafana link:
`https://console.aws.amazon.com/cloudwatch/home?region=us-east-1#logEventViewer:group=[[env]]-[[service]];stream=[[instance]]/app.log;start=[[__from:rfc3339]];end=[[__to:rfc3339]]`

- Expected Cloudwatch url:
`https://console.aws.amazon.com/cloudwatch/home?region=us-east-1#logEventViewer:group=prod-myservice;stream=i-01234567890123456/app.log;start=2019-10-22T22:00:00Z;end=2019-10-24T14:17:00Z`

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
